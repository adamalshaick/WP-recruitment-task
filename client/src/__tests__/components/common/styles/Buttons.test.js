import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import {
  Button,
  ButtonsCard
} from "../../../../components/common/styles/Buttons";

describe("render component", () => {
  it("renders button component", () => {
    const component = renderer.create(<Button />).toJSON();
    expect(component).toMatchSnapshot();
  });
  it("renders buttons card", () => {
    const component = renderer.create(<ButtonsCard />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
