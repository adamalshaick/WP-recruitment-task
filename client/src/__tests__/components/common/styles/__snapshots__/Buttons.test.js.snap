// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component renders button component 1`] = `
.c0 {
  line-height: 50px;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: black;
  width: 100%;
  text-align: center;
  -webkit-transition: all 0.25s;
  transition: all 0.25s;
}

.c0:hover {
  background-color: whitesmoke;
  border-bottom: blue solid 1px;
  color: blue;
}

<a
  className="c0"
/>
`;

exports[`render component renders buttons card 1`] = `
.c0 {
  background-color: white;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 500px;
  height: 50px;
  box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.25);
  position: relative;
}

@media (min-width:1000px) {
  .c0 {
    left: 50%;
    -webkit-transform: translateX(-50%);
    -ms-transform: translateX(-50%);
    transform: translateX(-50%);
  }
}

<span
  className="c0"
/>
`;
